NEWS.md

```diff
@@ -0,0 +1,25 @@
+ROhdsiWebApi 0.1.0
+=====================
+Note: this version has breaking changes because design changes and naming conventions.
+
+1. Generic design pattern: Based on discussions here https://github.com/OHDSI/ROhdsiWebApi/issues/37
```

Comment:
In general our OHDSI packages have two sections in NEWS (in line with CRAN expectations): Changes and Bug Fixes.